python/private/py_wheel.bzl

```diff
@@ -352,6 +414,33 @@ def _py_wheel_impl(ctx):
         ),
     ]
 
+def _inputs_to_package(actions, workspace_name, deps, links_subdir, linker):
+    filtered = []
+    for dep in deps:
+        files = dep[PyInfo].transitive_sources
+        imports = [imports.replace("\\", "/") for imports in dep[PyInfo].imports.to_list()]
+        for file in files.to_list():
+            filepath = workspace_name + "/" + file.path.replace("\\", "/")
+            for imp in imports:
+                if filepath.startswith(imp):
+                    link_path = filepath[len(imp):]
+                    if link_path.startswith("/"):
+                        link_path = link_path[1:]
+                    link_path = links_subdir + "/" + link_path
+                    link = actions.declare_directory(link_path) if file.is_directory else actions.declare_file(link_path)
```

Comment:
This action can probably be replaced with just actions.symlink() when the file isn't a directory?

Reply:
Trust me; I tried all the obvious solutions. CI would always fail for one of the platforms.